app/package.json

```diff
@@ -108,5 +108,8 @@
 		"vue-router": "4.0.12",
 		"vuedraggable": "4.1.0",
 		"wellknown": "0.5.0"
+	},
+	"dependencies": {
```

Comment:
All app dependencies are bundled in when we deploy the `@directus/app` package, so everything is a `devDependency` 👍🏻